src/blockstorage/blockstorage.cpp

```diff
@@ -796,15 +747,7 @@ bool FindBlockPos(CValidationState &state,
     {
         pos.nFile = 1;
         pos.nPos = nAddSize;
-        if (CheckDiskSpace(nAddSize))
-        {
-            nDBUsedSpace += nAddSize;
-            if (fPruneMode && nDBUsedSpace >= nPruneTarget)
```

Comment:
are you removing the old pruning method?  Its seems like you are not because the flag is still in addargs but a lot of that code is being removed...

Reply:
i did not remove the old pruning method. the code was moved/refactored into prune.cpp to make managing the pruning code easier overall mostly by reducing the amount of public api for pruning code. it is down to only a handful of functions.